tests/cypress/e2e/issues_prs2/issue_8872_delete_frame.js

```diff
@@ -0,0 +1,71 @@
+// Copyright (C) 2025 CVAT.ai Corporation
+//
+// SPDX-License-Identifier: MIT
+
+/// <reference types="cypress" />
+
+import { taskName } from '../../support/const';
+
+context('UI and job metadata work correctly when deleting frames', () => {
+    const chunkReloadPeriod = 100; // 100 ms
+    let defaultJobMetadataReloadPreiod;
+
+    describe('Attempt to delete any frame after repeated request to /data/meta/', () => {
+        before(() => {
+            cy.window().then((window) => {
+                defaultJobMetadataReloadPreiod = window.cvat.config.jobMetaDataReloadPeriod;
+                window.cvat.config.jobMetaDataReloadPeriod = chunkReloadPeriod;
+            });
+        });
+
+        it('Elapse job metadata reload period, delete a frame, validate UI state is and request body ', () => {
+            cy.intercept('GET', '/api/jobs/**/data/meta**').as('getMeta');
+            cy.intercept('PATCH', '/api/jobs/**/data/meta**').as('patchMeta');
+
+            cy.openTaskJob(taskName);
+            // Ensure first request is sent after loading the job
+            cy.wait('@getMeta');
+
+            const frameAlias = 'oldCurrentFrame';
+
+            cy.goToNextFrame(1);
+            cy.wait('@getMeta').then(() => {
+                // enqueue current frame number
+                // save it in context
+                cy.getCurrentFrameNumber(frameAlias);
+            });
```

Comment:
```suggestion
            cy.wait('@getMeta');
            
            let frameNum = null;
            cy.getCurrentFrameNumber().then(curNumber => frameNum = curNumber);
            
```

Using an alias to get frame number seems a bit overcomplicated. We can just wait a command to complete and save it in some local variable.